src/app/contact/contact.component.html

```diff
@@ -0,0 +1,339 @@
+        <p>93 Mau Than Street<br> Can Tho<br> Viet Nam<br> Phone +84 (0) 907 65 29 27<br> Email: support@susper.net<br> Board
+          of Directors: Phuc Hau Dang<br> Susper Ltd. is registered in Can Tho, Viet Nam.</p>
+      </div>
+    </div>
+    <!--Contact Us details ends-->
+  </div>
+  <!--Contact-sub-details ends here-->
+
+  <br>
+
+  <div class="contact-sub-details">
+    <div class="row">
+      <div class="col-lg-12 col-sm-12 contact-sub-details">
+        <p>Report a safety or abuse issue affecting our products.<br> If you know of a safety or abuse problem with any of Susper's
+          services, we'd like to hear about it right away. Please use our <a (click)="open()">contact</a> form to report the
+          issue.
+        </p>
+      </div>
+
+    </div>
+  </div>
+  <!--contact-sub-details ends here-->
+
+</div>
+<!--container ends here-->
+<footer>
+  <!--footer navigation bar goes here-->
+  <app-footer-navbar></app-footer-navbar>
+</footer>
+
+<modal #myModal>
+  <div>
+    <a (click)="close()" class="close-btn"> <i class="fa fa-times" aria-hidden="true" style="font-size: 2em"></i> </a>
+ 
+
+  </div>
+  <form class="form-horizontal" method="POST" action="http://formspree.io/office@fossasia.org">
+  <fieldset>
+
+  <!-- Form Name -->
+  <legend>Contact Form</legend>
+
+  <!-- Text input-->
+  <div class="form-group">
+    <label class="col-md-3 control-label" for="textinput">Your Name<sup>*</sup></label>  
+    <div class="col-md-9">
+    <input id="textinput" name="yourName" type="text" placeholder="Your Name" class="form-control input-md" required="" >
+      
+    </div>
+  </div>
+
+  <!-- Text input-->
+  <div class="form-group">
+    <label class="col-md-3 control-label"a for="textinput">Email<sup>*</sup></label>  
+    <div class="col-md-9">
+    <input id="textinput" name="emal-address" type="email" placeholder="Email Address" class="form-control input-md" required=""  >
+      
+    </div>
+  </div>
+
+  <!-- Select Basic -->
+  <div class="form-group row">
+    <label class="col-md-3 control-label" for="countrycode">Mobile Number<sup>*</sup></label>
+    <div class="col-md-2">
+
+      <select id="countrycode" name="countrycode" class="form-control">
+              <option data-countryCode="DZ" value="213">Algeria (+213)</option>
+              <option data-countryCode="AD" value="376">Andorra (+376)</option>
+              <option data-countryCode="AO" value="244">Angola (+244)</option>
+              <option data-countryCode="AI" value="1264">Anguilla (+1264)</option>
+              <option data-countryCode="AG" value="1268">Antigua &amp; Barbuda (+1268)</option>
+              <option data-countryCode="AR" value="54">Argentina (+54)</option>
+              <option data-countryCode="AM" value="374">Armenia (+374)</option>
+              <option data-countryCode="AW" value="297">Aruba (+297)</option>
+              <option data-countryCode="AU" value="61">Australia (+61)</option>
```

Comment:
Currently, we don't need to implement this right now. Just focus on simple feature. Let's have @mariobehling opinion in this.